src/core/cgroup.c

```diff
@@ -4070,6 +4070,8 @@ int unit_get_memory_peak(Unit *u, uint64_t *ret) {
         int r;
 
         assert(u);
+        assert(last);
+        assert(mem_attribute);
```

Comment:
These should not be in the first commit...

Reply:
Good catch. Thanks!